README.md

```diff
@@ -1,16 +1,130 @@
-# BabsECS
+# Babs-ECS
 [![Build Status](https://travis-ci.org/babsnbabs/babs-ecs.svg?branch=master)](https://travis-ci.org/babsnbabs/babs-ecs)
 
-Entity Component System implemented in C++, ideally with decent error handling and acceptable performance.
+A fast, simple, cross platform Entity Component System written for C++14.
```

Comment:
with `std::any` we're technicall C++17 . This is what I have at the top of the repository:

`C++17 Header-Only Fast, Simple, Entity-Component-System`

I don't prefer one or the other, just make sure the C++ version is correct and make them line up (you should have admin rights to update the repro description.)

Reply:
`A fast, simple, and cross platform C++17 header-only Entity Component System`